docs/scos/dev/feature-integration-guides/upcoming-release/cart-integration.md

```diff
@@ -0,0 +1,869 @@
+---
+title: Cart integration
+description: The guide describes the process of integrating the Cart feature into your project.
+last_updated: Jun 16, 2021
+template: feature-integration-guide-template
+originalLink: https://documentation.spryker.com/2021080/docs/cart-integration
+originalArticleId: baf4c073-efd0-4c8c-9fb5-9259fa8f2a6b
+redirect_from:
+  - /2021080/docs/cart-integration
+  - /2021080/docs/en/cart-integration
+  - /docs/cart-integration
+  - /docs/en/cart-integration
+---
+
+{% info_block infoBox %}
+
+This article describes how to add product variants and product images to an existing cart.
+{% endinfo_block %}
+
+## Prerequisites:
+
+Before starting make sure you are familiar with the concept of Spryker Super Attributes.
+
+## UI Changes:
+
+Cart now supports changing the items in the cart by modifying their attributes. If we have a wrong T-Shirt size in the cart we will be able to change it.
+
+Cart now also supports product images out of the box.
+![cart_product_images](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/feature-integration-guides/cart-integration.md/cart_product_images.png)
+
+If we have products with multiple super attributes we can now, narrowing-down in the cart.
```

Comment:
we can now do what?

Reply:
Haven't fixed.

Reply:
Based on the super attributes, you can select the needed product variant in cart.